packages/core-ethereum/src/index.ts

```diff
@@ -379,14 +407,14 @@ export default class HoprCoreEthereum extends EventEmitter {
           } else {
             // May fail due to out-of-commits, preimage-is-empty, not-a-winning-ticket
             // Treat those acked tickets as losing tickets, and remove them from the DB.
-            await boundMarkLosingAckedTicket(ticket)
+            await boundMarkLosingAckedTicket(Ethereum_AcknowledgedTicket.deserialize(ticket.serialize()))
             metric_losingTickets.increment()
           }
         }
 
         yield ticket.response
 
-        tickets = await boundGetAckdTickets({ channel })
+        tickets = await boundGetAckdTickets({ serdeChannel })
```

Comment:
I don't see where the key `serdeChannel` is used further down the call stack.

Reply:
defined as conversion at line 377